pyhit init file missing imports

## Bug Description
Add `import os` and `import subprocess` to init file for the pyhit module.

moose\python\pyhit\__init__.py:4: in <module>
    testdir = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', 'test'))
E   NameError: name 'os' is not defined

## Steps to Reproduce
When `from . import hit` fails.

## Impact
It is preventing me from getting my work done.

--- a/python/pyhit/__init__.py
+++ b/python/pyhit/__init__.py
@@ -1,3 +1,6 @@
+import os
+import subprocess
+
 try:
     from . import hit
 except:

